apps/cowswap-frontend/src/common/constants/banners.ts

```diff
@@ -0,0 +1,7 @@
+// Banner IDs for various dismissable banners across the app
+export const BANNER_IDS = {
+  DELEGATE: 'delegate-banner-v1',
```

Comment:
Can you do it camelCased like the others?

Ok, there's a mix of camelCase with/without underscore and one snake_case...
But let's at least not introduce a kebab-case to the mix 🍢